spec/lib/tasks/existing_firms_sign_up_task_spec.rb

```diff
@@ -1,107 +1,117 @@
+class StubInvitationHelper
+  def invitation_url(user)
+    "invitation for user #{user.id}"
+  end
+end
+
 module Tasks
   RSpec.describe ExistingFirmsSignUpTask do
     describe '#notify' do
-      let!(:bills_firm) { create :firm }
-      let!(:bill) { create :principal, email_address: 'bill@example.com', firm: bills_firm }
+      let!(:bills_firm) { create :firm, registered_name: 'Wright, Johnston & MacKenzie LLP' }
+      let!(:bill) do
+        principal = build :principal
+        principal.email_address =  'bill@example.com'
+        principal.firm = bills_firm
+        principal.first_name = 'Bill'
+        principal.last_name = 'Junior, the third'
+        principal.save!
+        principal
+      end
```

Comment:
Yeah, I didn't clean up. Most of the info was an overhang from the previous implementation using a factory. I added a real value from prod for the `last_name` line so that we checked that commas were escaped in the csv output. 

All of this made rubocop unhappy with the line length or splitting and it ended up as above when I should have cleaned it up with less.